src/main/resources/application-prod.yml

```
@@ -47,6 +47,9 @@ spring:
   data:
     redis:
       timeout: 30000ms
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)